admin/dw-options.php

```diff
@@ -75,20 +91,25 @@ function dw_render_options_page() {
 						</p>
```

Comment:
|`line 86`|
|---|
Maybe give this a more generic name now it's also used for non-tkm products?